package.json

```diff
@@ -19,15 +19,15 @@
     "angular-ui-bootstrap": "1.3.3",
     "async": "1.5.2",
     "blueimp-load-image": "2.6.1",
-    "bootstrap": "3.3.6",
+    "bootstrap": "3.3.7",
     "bootstrap-markdown": "2.10.0",
     "bootstrap-slider": "7.1.0",
     "closure-util": "1.15.1",
     "d3": "3.5.17",
     "moment": "2.15.2",
     "moment-timezone": "0.5.9",
     "ng-infinite-scroll": "1.3.0",
-    "photoswipe": "4.1.1",
+    "photoswipe": "4.1.2",
     "slick-carousel": "1.6.0",
```

Comment:
I think we no longer use slick-carousel since the new layout has been introduced (thumbnails are no longer displayed in a carousel) => perhaps we can drop this line.

Reply:
Yeah, I intend to do some cleaning once all the stuff is configured. Many dependecies are in the dev ones, while they should be in the default ones.